public/angular/translations/locale-en.json

```diff
@@ -19,6 +19,8 @@
   "rss": "RSS",
   "elmo": "Elmo",
   "facebook": "Facebook",
+  "smsgh": "SMS GH",
+  "Smsgh": "SMS GH",
```

Comment:
`grep -r Smsgh lib public models` doesn't turn up anything. Maybe the fault of `filters/capitalize`?